test/integration_test.js

```diff
@@ -258,19 +258,23 @@ describe('integration test', function() {
 
       describe('persistent output', function() {
         class BuildOnce extends Plugin {
+          constructor(node, options = {}) {
+            super(node, options);
+          }
           build() {
             if (!this.builtOnce) {
               this.builtOnce = true;
-              fs.writeFileSync(path.join(this.outputPath, 'foo.txt'), 'test');
+              this.output.writeFileSync('foo.txt', 'test');
             }
           }
         }
```

Comment:
We shouldn't change the meaning of this test, we should add additional tests that test the new behaviors. The vast majority of broccoli-plugin consumers will continue to use `fs` directly (at least for a while)...

Reply:
fixing